src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -215,6 +224,11 @@ public IChangesObservable<OperationStatusChange> ForAllOperations()
             return taskedObservable;
         }
 
+        private void ThrowOperationsNodeTagNeeded()
+        {
+            throw new InvalidOperationException("Changes API must be provided a node tag in order to track node-specific operations.");
```

Comment:
In the direct talk we found that for `WaitForCompletionAsync()` calls we ensure it internally here:
https://github.com/ravendb/ravendb/blob/3894f2909305381ff070a640a1240a6dfb709703/src/Raven.Client/Documents/Operations/MaintenanceOperationExecutor.cs#L84
right?

What about the below:
https://github.com/ravendb/ravendb/blob/3894f2909305381ff070a640a1240a6dfb709703/src/Raven.Client/Documents/Operations/MaintenanceOperationExecutor.cs#L95
where we don't pass node tag?

Reply:
It has been changes as well https://github.com/ravendb/ravendb/pull/18417/files#diff-80a3634d824780e0a5e6fa6765ef56652c0c78c69f7f9df18f15f05e628fabe9R95

Reply:
OK